libraries/ms-drivers/inc/ltc6811.h

```diff
@@ -66,13 +69,43 @@ typedef struct {
 } _PACKED LtcAfeConfigRegisterData;
 static_assert(sizeof(LtcAfeConfigRegisterData) == 6, "LtcAfeConfigRegisterData must be 6 bytes");
 
+// COMM Register, refer to LTC6803 datasheet page 31, Table 15
+typedef struct {
+  // COMM0
+  uint8_t icom0 : 4;
+  uint8_t d0 : 8;
+  uint8_t fcom0 : 4;
+
+  uint8_t icom1 : 4;
+  uint8_t d1 : 8;
+  uint8_t fcom1 : 4;
+
+  uint8_t icom2 : 4;
+  uint8_t d2 : 8;
+  uint8_t fcom2 : 4;
+} _PACKED LtcAfeCommRegisterData;
+static_assert(sizeof(LtcAfeCommRegisterData) == 6, "LtcAfeCommRegisterData must be 6 bytes");
+
 // CFGR packet
 typedef struct {
   LtcAfeConfigRegisterData reg;
 
   uint16_t pec;
 } _PACKED LtcAfeWriteDeviceConfigPacket;
 
+// WRCOMM + mux pin
+typedef struct {
+  uint8_t wrcomm[4];
+  LtcAfeCommRegisterData reg;
+  uint8_t pec;
+} _PACKED LtcAfeWriteCommRegPacket;
+
+// STMCOMM + clock cycles
+typedef struct {
+  uint8_t stcomm[4];
+  uint8_t clk[3];
+} _PACKED LtcAfeSendCommRegPacket;
```

Comment:
I would say make 2 defines for the amount of bytes for each of these (a define for the size of a command would be nice because we just write the magic number everywhere) so that you don't have to use SIZEOF_ARRAY